rpc/wrtc_signaling_answerer.go

```diff
@@ -144,7 +144,8 @@ func checkExceptionalError(err error) error {
 				strings.Contains(s.Message(), "too_many_pings") ||
 				// RSDK-3025: Cloud Run has a max one hour timeout which will terminate gRPC
 				// streams, but leave the underlying connection open.
-				strings.Contains(s.Message(), "upstream max stream duration reached"))) {
+				strings.Contains(s.Message(), "upstream max stream duration reached") ||
+				strings.Contains(s.Message(), "server closed the stream without sending trailers"))) {
```

Comment:
This new predicate swallows the annoying, ubiquitous, and seemingly meaningless log:
```
11/18/2024, 12:23:35 PM warn rdk.networking.signaler.external rpc/wrtc_signaling_answerer.go:242 error communicating with signaling server error rpc error: code = Internal desc = server closed the stream without sending trailers
```